ui/config/webpack.config.js

```diff
@@ -158,14 +167,12 @@ module.exports = function(webpackEnv) {
       // There will be one main bundle, and one file per asynchronous chunk.
       // In development, it does not produce real files.
       filename: isEnvProduction
-        ? 'static/js/[name].js'
-        : isEnvDevelopment && 'static/js/bundle.js',
+        ? 'static/js/[name].[hash:6].js'
+        : 'static/js/[name].js',
```

Comment:
The `hash` is a checksum, so it provides cache breaking.  I'm not sure why webpack doesn't include it by default (I thought it did, but who knows).

The `name` here is necessary as now we have separate bundles for each page.